contrib/NeonVM/neonvm.c

```diff
@@ -0,0 +1,820 @@
+{
+    error("not implemented");
+}
+
+void exec_STORES()
+{
+    error("not implemented");
+}
+
+void exec_STOREA()
+{
+    error("not implemented");
+}
+
+void exec_STORED()
+{
+    error("not implemented");
+}
+
+void exec_STOREP()
+{
+    error("not implemented");
+}
+
+void exec_NEGN()
+{
+    error("not implemented");
+}
+
+void exec_ADDN()
+{
+    error("not implemented");
+}
+
+void exec_SUBN()
+{
+    error("not implemented");
+}
+
+void exec_MULN()
+{
+    error("not implemented");
+}
+
+void exec_DIVN()
+{
+    error("not implemented");
+}
+
+void exec_MODN()
+{
+    error("not implemented");
+}
+
+void exec_EXPN()
+{
+    error("not implemented");
+}
+
+void exec_EQB()
+{
+    error("not implemented");
+}
+
+void exec_NEB()
+{
+    error("not implemented");
+}
+
+void exec_EQN()
+{
+    error("not implemented");
+}
+
+void exec_NEN()
+{
+    error("not implemented");
+}
+
+void exec_LTN()
+{
+    error("not implemented");
+}
+
+void exec_GTN()
+{
+    error("not implemented");
+}
+
+void exec_LEN()
+{
+    error("not implemented");
+}
+
+void exec_GEN()
+{
+    error("not implemented");
+}
+
+void exec_EQS()
+{
+    error("not implemented");
+}
+
+void exec_NES()
+{
+    error("not implemented");
+}
+
+void exec_LTS()
+{
+    error("not implemented");
+}
+
+void exec_GTS()
+{
+    error("not implemented");
+}
+
+void exec_LES()
+{
+    error("not implemented");
+}
+
+void exec_GES()
+{
+    error("not implemented");
+}
+
+void exec_EQA()
+{
+    error("not implemented");
+}
+
+void exec_NEA()
+{
+    error("not implemented");
+}
+
+void exec_EQD()
+{
+    error("not implemented");
+}
+
+void exec_NED()
+{
+    error("not implemented");
+}
+
+void exec_EQP()
+{
+    error("not implemented");
+}
+
+void exec_NEP()
+{
+    error("not implemented");
+}
+
+void exec_ANDB()
+{
+    error("not implemented");
+}
+
+void exec_ORB()
+{
+    error("not implemented");
+}
+
+void exec_NOTB()
+{
+    error("not implemented");
+}
+
+void exec_INDEXAR()
+{
+    error("not implemented");
+}
+
+void exec_INDEXAW()
+{
+    error("not implemented");
+}
+
+void exec_INDEXAV()
+{
+    error("not implemented");
+}
+
+void exec_INDEXAN()
+{
+    error("not implemented");
+}
+
+void exec_INDEXDR()
+{
+    error("not implemented");
+}
+
+void exec_INDEXDW()
+{
+    error("not implemented");
+}
+
+void exec_INDEXDV()
+{
+    error("not implemented");
+}
+
+void exec_INA()
+{
+    error("not implemented");
+}
+
+void exec_IND()
+{
+    error("not implemented");
+}
+
+void exec_CALLP(struct tagTExecutor *self)
+{
+    uint32_t val = (self->object->code[self->ip+1] << 24) | (self->object->code[self->ip+2] << 16) | (self->object->code[self->ip+3] << 8) | self->object->code[self->ip+4];
+    self->ip += 5;
+    const unsigned char *func = self->object->strtable[val];
+
+    if (strcmp((const char*)func, "print") == 0) {
+        const char *s = pop(&self->stack);
+        printf("%s\n", s);
+    } else {
+        error("exec_CALLP(): Invalid function call");

```

Comment:
Helpful here to print out what the requested `func` actually is (for diagnostic purposes).